### PR TITLE
CI: Update the `lint commits` error message to be more user friendly

### PR DESCRIPTION
**Pull request by bgianfo**

### CODE REVIEW

File: .github/workflows/lintcommits.yml [hunk 1]
```diff
@@ -77,4 +77,4 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.BUGGIEBOT }}
         with:
-          msg: "One or more of the commits in this PR do not match the [code submission policy](https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#code-submission-policy), please check the `lint_commits` CI job for more details."
+          msg: "Hello!\n\nOne or more of the commits messages in this PR do not match the SerenityOS [code submission policy](https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#code-submission-policy), please check the `lint_commits` CI job for more details on which commits were flagged and why.\nPlease do not close this PR and open another, instead modify your commit message(s) with [git commit --amend][https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/changing-a-commit-message] and force push those changes to update this PR."
```
Comment (alimpfard) [inline .github/workflows/lintcommits.yml:80]: Markdown mishap on `git commit --amend`.

Comment (bgianfo) [reply .github/workflows/lintcommits.yml]: whoops... fixed, thanks!

**Comment by alimpfard** [comment]
I just noticed..."commits messages".

**Comment by bgianfo** [comment]
> I just noticed..."commits messages".

Yes, fixed!

**Review by alimpfard — APPROVED** [review]

**Review by IdanHo — APPROVED** [review]
LGTM, a little bit long, but should be good.

**Pull request merged**